### DISCUSSION TITLE
Commercial use of Tasmota firmware

### DISCUSSION THREAD

**Discussion by MR91** [arendst/Tasmota#15291] [General]
Hi there,

one of the companies I work with is planning to sell devices flashed with Tasmota firmware into them. I was wondering wether we are allowed to promote the Tasmota functionality (something like "Tasmota inside" or god knows what)?

From the GPL 3.0 License, I can't find any reason as to why we would not be allowed to ship Tasmota on our devices. However, the question about promoting the compatibility remains.

Would love to get any information on this!

Best regards

**Comment by barbudor** [comment]
Hi
There are already products shipped with Tasmota inside and their vendors are actually promoting it (you can have a look of those we are aware of [here](https://templates.blakadder.com/preflashed.html))
The GPL 3.0 do not prevent such usage as far as you are not blocking any of your customer to be able upgrade freely the device.
For more details you should discuss this with Theo @arendst

**Comment by MR91** [comment]
Hey @barbudor,

Thanks for your reply! I was aware of the Athom devices, however from the looks of their website I was uncertain wether this is an officially supported product.

Thanks for the information, so my assumption was correct an we can carry on!

Best regards

**Reply by barbudor** [reply]
It all depends what you mean by "supported product".
It is definitively an Athom Product supported by Athom.
